pkg/crc/services/dns/dns_windows.go

```diff
@@ -30,7 +29,7 @@ func runPostStartForOS(serviceConfig services.ServicePostStartConfig) error {
 
 	time.Sleep(2 * time.Second)
 
-	if !contains(getInterfaceNameserverValues(networkInterface), serviceConfig.IP) {
+	if !crcos.Contains(getInterfaceNameserverValues(networkInterface), serviceConfig.IP) {
```

Comment:
then is this not in error?

```
if !crcos.Contains(getInterfaceNameserverValues(networkInterface), serviceConfig.IP) {
```

the order for the parameters have changed? https://github.com/crc-org/crc/pull/3765/files#r1270651088 Let's stay consistent.

Reply:
`dns_windows.go` was using `contains(s []string, e string) bool` and `clusteroperator.go` used `contains(value string, list []string) bool`,  so the order will change in one of the instances.

Reply:
Check

Can we in that case also more clearly describe the parameters:

```
func Contains(s []string, e string) bool {
```

with actual 'words' instead of abbreviated identifiers. https://github.com/crc-org/crc/pull/3765#discussion_r1270661563

`s` = `strings` and `e` = `expected`?